t/opcode-coverage.neon

```diff
@@ -0,0 +1,697 @@
+FUNCTION les(): Boolean
+    RETURN "A" <= "A"
+END FUNCTION
+
+#    GES,        // compare greater equal string
+FUNCTION ges(): Boolean
+    RETURN "A" >= "A"
+END FUNCTION
+
+#    EQY,        // compare equal bytes
+FUNCTION eqy(): Boolean
+    RETURN hexbytes = hexbytes
+END FUNCTION
+
+#    NEY,        // compare unequal bytes
+FUNCTION ney(): Boolean
+    RETURN hexbytes <> hexbytes
+END FUNCTION
+
+#    LTY,        // compare less bytes
+FUNCTION lty(): Boolean
+    RETURN hexbytes < hexbytes
+END FUNCTION
+
+#    GTY,        // compare greater bytes
+FUNCTION gty(): Boolean
+    RETURN hexbytes > hexbytes
+END FUNCTION
+
+#    LEY,        // compare less equal bytes
+FUNCTION ley(): Boolean
+    RETURN hexbytes <= hexbytes
+END FUNCTION
+
+#    GEY,        // compare greater equal bytes
+FUNCTION gey(): Boolean
+    RETURN hexbytes >= hexbytes
+END FUNCTION
+
+#    EQA,        // compare equal array
+FUNCTION eqa(): Boolean
+    RETURN array = array
+END FUNCTION
+
+#    NEA,        // compare unequal array
+FUNCTION nea(): Boolean
+    RETURN array <> array
+END FUNCTION
+
+#    EQD,        // compare equal dictionary
+FUNCTION eqd(): Boolean
+    RETURN dictionary = dictionary
+END FUNCTION
+
+#    NED,        // compare unequal dictionary
+FUNCTION ned(): Boolean
+    RETURN dictionary <> dictionary
+END FUNCTION
+
+#    EQP,        // compare equal pointer
+FUNCTION eqp()
+    IF NIL = NIL THEN
+    END IF
+END FUNCTION
+
+#    NEP,        // compare unequal pointer
+FUNCTION nep()
+    IF NIL <> NIL THEN
+    END IF
+END FUNCTION
+
+#    EQV,        // compare equal voidptr
+FUNCTION eqv()
+    VAR p1: Handle
+    p1 := NIL
+    IF p1 = NIL THEN
+    END IF
+END FUNCTION
+
+#    NEV,        // compare unequal voidptr
+FUNCTION nev()
+    VAR p1: Handle
+    p1 := NIL
+    IF p1 <> NIL THEN
+    END IF
+END FUNCTION
+
+#    ANDB,       // and boolean
+FUNCTION andb(): Boolean
+    RETURN TRUE AND FALSE
+END FUNCTION # Does not generate opcode in Neon binaries.
+
+#    ORB,        // or boolean
+FUNCTION orb(): Boolean
+    RETURN TRUE OR FALSE
+END FUNCTION # Does not generate opcode in Neon binaries.
+
+#    NOTB,       // not boolean
+FUNCTION notb(): Boolean
+    RETURN NOT TRUE
+END FUNCTION
+
+#    INDEXAR,    // index array for read
+FUNCTION indexar(): Number
+    RETURN array[0]
+END FUNCTION
+
+#    INDEXAW,    // index array for write
+FUNCTION indexaw()
+    array[42] := 42
+END FUNCTION
+
+#    INDEXAV,    // index array value
+FUNCTION indexav(): Number
+    FUNCTION a(): Array<Number>
+        RETURN array
+    END FUNCTION
+    RETURN a()[1]
+END FUNCTION
+
+#    INDEXAN,    // index array value, no exception
+FUNCTION indexan(): String
+    RETURN rec().field
+END FUNCTION
+
+#    INDEXDR,    // index dictionary for read
+FUNCTION indexdr(): Number
+    RETURN dictionary["One"]
+END FUNCTION
+
+#    INDEXDW,    // index dictionary for write
+FUNCTION indexdw()
+    dictionary["Neon"] := 20.1797
```

Comment:
What is with this weird decimal value here? Don't confuse the issue by using random values. Just assign something obvious like 5.

Reply:
It was the atomic weight of Neon!  :)

Reply:
Changed to "42".